### ISSUE TITLE
The possibility of saving data store without the need of user login

### ISSUE THREAD

**Issue by Michaelgimii** [CrocoBlock/suggestions#6908]
Hi team.

Currently, when we use the data store module, the value is not saved when the user is not logged in and likes for example. Please add this option so that even users who are not logged in, their data is saved. Like many websites that actually collect votes from their users without users being logged in.

**Comment by Lonsdale201** [comment]
Yes, you can do it, use other storage type not the user metadata. 😉

**Comment by Michaelgimii** [comment]
Hi @Lonsdale201, I apologize for the delay. So should I set the Store type on the cookie?

**Comment by aalexandr21** [comment]
Yes, you can use any other storage types, because only the User Meta store works with logged-in users
